src/main/java/friday/logic/commands/FindCommand.java

```diff
@@ -27,6 +28,7 @@ public FindCommand(NameContainsKeywordsPredicate predicate) {
 
     @Override
     public CommandResult execute(Model model) {
+        assert true;
```

Comment:
Instead of `assert true`, perhaps you can replace it with something like `assert !isNull(predicate)`?